components/dialog/index.css

```diff
@@ -142,13 +130,13 @@ governing permissions and limitations under the License.
 .spectrum-Dialog .spectrum-Dialog-divider {
   grid-area: divider;
   inline-size: 100%;
-  margin-top: var(--spectrum-dialog-confirm-divider-margin-top);
-  margin-bottom: var(--spectrum-dialog-confirm-divider-margin-bottom);
+  margin-top: var(--mod-dialog-confirm-padding-200,var(--spectrum-dialog-confirm-padding-200));
+  margin-bottom: var(--mod-dialog-confirm-padding-300,var(--spectrum-dialog-confirm-padding-300));
```

Comment:
Could these use the logical properties `margin-block-start` and `margin-block-end`?

Reply:
I have used `--spectrum-dialog-confirm-divider-block-spacing-start`and
`--spectrum-dialog-confirm-divider-block-spacing-end`